src/java/picard/vcf/LiftoverVcf.java

```diff
@@ -147,7 +147,8 @@ public static void main(final String[] args) {
             final Interval source = new Interval(ctx.getContig(), ctx.getStart(), ctx.getEnd(), false, ctx.getContig() + ":" + ctx.getStart() + "-" + ctx.getEnd());
             final Interval target = liftOver.liftOver(source, 1.0);
 
-            if (target == null) {
+            // if the target is null OR (the target is reverse complemented AND the variant is an indel), then we cannot lift it over
+            if (target == null || (target.isNegativeStrand() && (ctx.isMixed() || ctx.isIndel()))) {
                 rejects.add(new VariantContextBuilder(ctx).filter(FILTER_CANNOT_LIFTOVER).make());
```

Comment:
I think it would be good to have a better description of why we cannot liftover this variant, to separate it from the other reasons.